docs/0.1/creating_products.md

```diff
@@ -0,0 +1,315 @@
+      `-url` option on the `grid` command line.
+
+      ```
+      root@gridd-alpha:/# export GRID_DAEMON_ENDPOINT="http://localhost:8080"
+      ```
+
+   c. Set `GRID_SERVICE_ID` to the full circuit ID, in the format
+      <code><i>circuitID</i>::<i>serviceID</i></code> (such as
+      `01234-ABCDE::xyZZ`). This environment variable replaces the
+
+     `--service_id` option on the `grid` command line.
+
+      ```
+      root@gridd-alpha:/# export GRID_SERVICE_ID="01234-ABCDE::xyZZ"
+      ```
+
+      Tip: See [Prerequisites](#prerequisites) for the commands to display
+      circuit and service IDs.
+
+### Define and Create a Product
+
+This section has two steps. First, create a product definition file in YAML
+format. Then run `grid product create` to create the product based on this
+definition.
+
+**Note**: Each product requires an organization ID (the owner) and at least one
+agent with permission to create, update, and delete the product. See
+[Prerequisites](#prerequisites) for the commands to display the existing
+organizations and agents.
+
+{:start="4"}
+
+4. Create a product definition file, in YAML format, that specifies the
+   following information:
+
+   * Product identification type (such as GS1)
+   * Product ID (such as a GTIN)
+   * Owner (organization ID)
+   * Set of product properties (each with a name, data type, and value
+     that conforms to the data type)
+     <br><br>
+
+   For example, you can use `cat` to create the file `product.yaml`, using the
+   following contents.
+
+   ```
+   root@gridd-alpha:/# cat > product.yaml
+   - product_type: "GS1"
+     product_id: "723382885088"
+     owner: "314156"
+     properties:
+       - name: "species"
+         data_type: "STRING"
+         string_value: "tuna"
+       - name: "length"
+         data_type: "NUMBER"
+         number_value: 22
+       - name: "maximum_temperature"
+         data_type: "NUMBER"
+         number_value: 5
+       - name: "minimum_temperature"
+         data_type: "NUMBER"
+         number_value: 0
+   ```
+
+   Tip: Enter CTRL-C to exit the file and save the contents.
+
+1. Add the new product by using the `grid product create` command to specify the
+   definition in the `product.yaml` file.
+
+   ```
+   root@gridd-alpha:/# grid product create product.yaml
+   ```
+
+   This command creates and submits a transaction to add the product data to the
+   distributed ledger. If the transaction is successful, all other nodes in the
+   circuit can view the product data.
+
+   **Note**: This command does not display any output. Instead, check the log
+   messages on the console (or the terminal window where you started the Grid
+   Docker environment) for the success or failure of this operation.
+
+### Display Product Information
+
+{:start="6"}
+
+6. List all existing products to verify that the new product has been added.
+
+   ```
+   root@gridd-alpha:/# grid product list
+   Product Id: "723382885088"
```

Comment:
This has weird formatting in the website

Reply:
(same for the repeated instance below)

Reply:
@agunde406 I'm not seeing any format weirdness in my local tests with 0.0.0.0:4000.  Can you explain what you're seeing? (Or attach a screen shot?)

Reply:
Fixed the spacing weirdness. (It was caused by tabs on some of the lines.)